src/components/hotkeys/hotkeys.service.ts

```diff
@@ -143,14 +141,14 @@ export function Hotkey(key, description: string, options?: any) {
 
 @Injectable()
 export class HotkeysService {
-  constructor(private ngZone: NgZone) {}
-
   hotkeys = hotkeys;
   suspend = _suspend;
   activate = _activate;
   deregister = _deregister;
   changeEvent = hotkeyChangedSource.asObservable();
 
+  constructor(private ngZone: NgZone) {}
```

Comment:
Decorators work because of this: https://github.com/Hypercubed/ngx-ui/blob/d73c0550f0eab83257b0fde5e50fa9415ffdcd90/src/components/hotkeys/hotkeys.service.ts#L131

You can test this in the demo by using the theme switching hotkeys.